x/alliance/client/cli/gov.go

```diff
@@ -35,31 +36,11 @@ func CreateAlliance() *cobra.Command {
 
 			denom := args[0]
 
-			rewardWeight, err := sdk.NewDecFromStr(args[1])
-			if err != nil {
-				return err
-			}
-
-			rewardWeightMin, err := sdk.NewDecFromStr(args[2])
-			if err != nil {
-				return err
-			}
-
-			rewardWeightMax, err := sdk.NewDecFromStr(args[3])
-			if err != nil {
-				return err
-			}
-
-			takeRate, err := sdk.NewDecFromStr(args[4])
-			if err != nil {
-				return err
-			}
-
-			rewardChangeRate, err := sdk.NewDecFromStr(args[5])
-			if err != nil {
-				return err
-			}
-
+			rewardWeight := sdkmath.LegacyMustNewDecFromStr(args[1])
```

Comment:
is there no equivalent interface? this will panic in an ugly way on cli if user typos

Reply:
resolved in https://github.com/Switcheo/alliance/pull/2/commits/df9dbe0794875ad98bbd8ffee31a9843ebe0fafe